code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -392,6 +455,26 @@ GLOBAL_LIST_INIT(psychic_sense_blacklist, typecacheof(list(/turf/open, /obj/mach
 /obj/effect/psyphoza_spores/proc/upgrade()
 	upgrades = min(upgrade_limit, upgrades+1)
 
+//Keybind for sense
+/datum/keybinding/mob/psychic_sense
+	key = "Shift-Space"
+	name = "psuchic_sense"
+	full_name = "Psychic Sense"
+	description = "Activates psyphoza psychic sense"
+	keybind_signal = COMSIG_SPECIES_ACTION_PRIMARY
+	///Ref to sense
+	var/datum/action/item_action/organ_action/psychic_highlight/PH
+
+/datum/keybinding/mob/psychic_sense/down(client/user)
+	. = ..()
+	if(.)
+		return
+	if(!user.mob) 
+		return
+	PH = PH || locate(/datum/action/item_action/organ_action/psychic_highlight) in user.mob.actions
```

Comment:
this should forward to their species datum and then be handled on the species

Reply:
https://www.youtube.com/watch?v=6AaKkGrTqjE